n3fit/src/n3fit/hyper_optimization/mongofiletrials.py

```diff
@@ -154,6 +155,7 @@ def __init__(
             f"mongo://{self.db_host}:{self.db_port}/{self._process_db_name(self.db_name)}/jobs"
         )
         self.workers = []
+        self.job_output_name = replica_path.parts[-3]
```

Comment:
```suggestion
        self.output_folder_name = replica_path.parts[-3]
```

Perhaps calling it `output_folder_name` or `runcard_name` makes this a bit clearer.